src/lib/flags.js

```diff
@@ -19,6 +19,7 @@ export const initFlags = () => {
 
 const flagsList = () => {
   flag('switcher', true)
-  flag('settings.partial-desktop-sync.show-synced-folders-selection'),
-    flag('settings.permissions-dashboard', true)
+  flag('settings.partial-desktop-sync.show-synced-folders-selection')
+  flag('settings.permissions-dashboard', true)
+  flag('settings.flagship-mode')
```

Comment:
could it be unified?

flag('name', true) or flag('name') 🙏

Reply:
not sure what you mean sorry? some flags we don't want to activate them from the front-end, but from the stack

Reply:
Yep, there is a different meaning between one arg vs 2 args: 
https://github.com/cozy/cozy-libs/blob/master/packages/cozy-flags/src/flag.js#L9-L17

One is a getter, two is a setter. 

It looks that we're setting `   flag('settings.permissions-dashboard', true) ` this flag to true, so if we publish a new version of the Settings app, by default everyone will have access to this page. I think this is not what we want (cc @Noemie21)

Reply:
I can't speak for the other flags, I just know that flagship-mode must definitely not be enabled by defualt

Reply:
> One is a getter, two is a setter.

In that case, we want to have a setter, then.

I think we should do:
```
flag('settings.permissions-dashboard', false)
  flag('settings.flagship-mode', false)
```

Reply:
This would disable the flags coming from the stack I think?
flag('anything') will be false by default unless activated in features

Reply:
ok, i start to understand.

The syntax of this feature is very confusing 😠

Reply:
It's merged : https://github.com/cozy/cozy-settings/pull/462